src/main/webapp/js/instructorFeedbackEdit/feedbackPath.js

```diff
@@ -0,0 +1,70 @@
+$(document).ready(function() {
+    FeedbackPath.attachEvents();
+});
+
+var FeedbackPath = {
+    attachEvents: function() {
+        var allDropdownOptions = $('.feedback-path-dropdown-option');
+        FeedbackPath.attachEventsForAllOptions(allDropdownOptions);
```

Comment:
Is the "class" name `FeedbackPath` needed? In this closure just the function name should be enough, right?